### DISCUSSION TITLE
Turn off field nesting from multi-step

### DISCUSSION THREAD

**Discussion by Freeeon** [formkit/formkit#899] [Ideas]
I create a big form with multistep plugin. But every step save values in nested object with name of the step.
I want to be able to disable this behavior. Because if I want nested structure, I can use Groups. 

In my situation, I have entities, that I break into several step for the convenience of people. But I don`t want fix structure back after multi-step

Example
```html
<FormKit v-model='data.form' type="multi-step" tab-style="tab">
  <FormKit type="step" name="stepOne">
    <FormKit type="text" name="name" validation="required" />
  </FormKit>
  <FormKit type="step" name="stepTwo">
    <FormKit type="textarea" name="story" validation="required" />
  </FormKit>
</FormKit>
``` 

```javascript
// Bad behavior for me
{
  stepOne: {
    name: "Test"
  },
  stepTwo: {
    story: "Nested =("
  }
}

// Preferred 
{
  name: "Test",
  story: "Not Nested =)"
}
```

**Comment by fenilli** [comment]
That is one of the limitations of the multi-step plugin, we talked about havint this feature in a bunch of ways but nothing concrete about it yet.
